lib/Models/Catalog/Ows/WebMapServiceCapabilitiesStratum.ts

```diff
@@ -110,6 +110,13 @@ export default class WebMapServiceCapabilitiesStratum extends LoadableStratum(
       );
   }
 
+  @computed
+  get layerLimit() {
+    if (isDefined(this.capabilities?.Service?.LayerLimit)) {
+      return parseInt(this.capabilities?.Service?.LayerLimit);
```

Comment:
Doesn't the previous line guarantee that `this.capabilities.Service.LayerLimit` is defined, so the question marks can be omitted?